### DISCUSSION TITLE
how to train pronunciation of individual words

### DISCUSSION THREAD

**Discussion by pivolan** [coqui-ai/TTS#2849] [General Q&A]
I have trained models in German, vits, and the pronunciation of some words and phrases needs to be corrected. What is the best way to finetune the model for some specific words? And is it possible to train the pronunciation of a word by another speaker? For example lock vocoder layer, and finetune melgram model, or try to convert with voice-to-voice sources, bring them to the main speaker, and run finetune as usual?
During finetune I plan to mix the new dataset with the old one in different proportions.
Have you had such practice?

**Comment by thorstenMueller** [comment]
Hi @pivolan ,
have you trained your model using eSpeak(-ng) as phonemizer?
Maybe adjusting it's dictionary might help. Not sure if this is helpful in your case, but i've created a tutorial video about that on my Thorsten-Voice channel.
https://youtu.be/493xbPIQBSU

**Reply by pivolan** [reply]
Yes, I used espeak as phonemizer. Thanks, it looks very helpful. And thank you for you dataset, it is awesome!
